shell/components/form/NameNsDescription.vue

```diff
@@ -223,16 +223,14 @@ export default {
     namespaces() {
       const currentStore = this.$store.getters['currentStore'](this.namespaceType);
       const namespaces = this.namespacesOverride || this.$store.getters[`${ currentStore }/all`](this.namespaceType);
+      const filterNamespace = this.$store.getters['allNamespaces'];
 
       const filtered = namespaces.filter( this.namespaceFilter || ((namespace) => {
         // By default, include the namespace in the dropdown.
         let out = true;
 
-        if (this.currentProduct?.hideSystemResources) {
-          // Filter out the namespace
-          // if it is a system namespace or if it is managed by
-          // Fleet.
-          out = !namespace.isSystem && !namespace.isFleetManaged;
```

Comment:
Why change the behavior of `hideSystemResources`? I don't think this component should be an exception to hiding system resources without a strong justification: If namespaces need to be hidden from the project/namespace list and hidden in the namespace filter, I would expect that they also need to be hidden here. 

Why are we no longer filtering out fleet-managed namespaces here, but are still filtering them elsewhere?